script/upgrades/MU01/governance/MU01-CGP-Phase2.sol

```diff
@@ -238,6 +256,153 @@ contract MU01_CGP_Phase2 is ICGPBuilder, GovernanceScript {
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(medianDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, medianDeltaBreaker, 3)
+        )
+      );
+    }
+
+    // Add the Value Delta Breaker to the breaker box with the trading mode '3' -> trading halted
+    if (breakerBox != address(0) || BreakerBox(breakerBox).breakerTradingMode(valueDeltaBreaker) == 0) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          breakerBox,
+          abi.encodeWithSelector(BreakerBox(0).addBreaker.selector, valueDeltaBreaker, 3)
+        )
+      );
+    }
+
+    // Set rate feed dependency for cEUR/USDC to cUSD/USDC.
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        breakerBox,
+        abi.encodeWithSelector(
+          BreakerBox(0).setRateFeedDependencies.selector,
+          cEURUSDCConfig.referenceRateFeedID,
+          Arrays.addresses(cUSDUSDCConfig.referenceRateFeedID)
+        )
+      )
+    );
+
+    // Set rate feed dependency for cBRL/USDC to cUSD/USDC.
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        breakerBox,
+        abi.encodeWithSelector(
+          BreakerBox(0).setRateFeedDependencies.selector,
+          cBRLUSDCConfig.referenceRateFeedID,
+          Arrays.addresses(cUSDUSDCConfig.referenceRateFeedID)
+        )
+      )
+    );
+
+    // Enable Median Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isMedianDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              medianDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Enable Value Delta Breaker for rate feeds
+    for (uint256 i = 0; i < poolConfigs.length; i++) {
+      if (poolConfigs[i].isValueDeltaBreakerEnabled) {
+        transactions.push(
+          ICeloGovernance.Transaction(
+            0,
+            breakerBox,
+            abi.encodeWithSelector(
+              BreakerBox(0).toggleBreaker.selector,
+              valueDeltaBreaker,
+              poolConfigs[i].referenceRateFeedID,
+              true
+            )
+          )
+        );
+      }
+    }
+
+    // Set BreakerBox address in SortedOracles
+    transactions.push(
+      ICeloGovernance.Transaction(
+        0,
+        contracts.celoRegistry("SortedOracles"),
+        abi.encodeWithSelector(SortedOracles(0).setBreakerBox.selector, breakerBox)
+      )
+    );
+  }
+
+  function proposal_configureMedianDeltaBreaker() public {
```

Comment:
We're missing the configuration of the value delta breaker for the new pools that were added. The new config for both of these pools is set to use this breaker. If Nadiem confirms that they should be using the value delta breaker, this should be added.

Reply:
I think new pools will be configured with medianDeltaBreaker